source/common/http/http1/codec_impl.h

```diff
@@ -331,8 +333,8 @@ class ServerConnectionImpl : public ServerConnection, public ConnectionImpl {
  */
 class ClientConnectionImpl : public ClientConnection, public ConnectionImpl {
 public:
-  ClientConnectionImpl(Network::Connection& connection, ConnectionCallbacks& callbacks);
-
+  ClientConnectionImpl(Network::Connection& connection, ConnectionCallbacks& callbacks,
+                       uint32_t max_response_headers_kb = 0x7fffffff);
```

Comment:
Is this really the default you want? Don't you want this limited? If this is the default, please add a comment on why.

Reply:
If we want this PR to be a no-op for response headers, I think the response header limits should be whatever the old compiler defaults were.